File: Patterns/Kotlin/Triengle.kt
// Triangle
/*

                 *
               * * *
             * * * * *
           * * * * * * *

*/
fun main() {

    var count = 8
    for (i in 1..count / 2) {
        for (j in 1..count - 1) {
            if (j > count / 2 - i && j < count / 2 + i) {
                System.out.print("* ")
            } else {
                System.out.print("  ")
            }
        }
        System.out.println()
    }
}

File: Patterns/C++/triengle.cpp
#include <iostream>
// Triangle
/*

                 *
               * * *
             * * * * *
           * * * * * * *

*/
using namespace std;
int main()
{

    int count = 8;
    for (int i = 1; i <= count / 2; i++)
    {
        for (int j = 1; j < count; j++)
        {
            if (j > count / 2 - i && j < count / 2 + i)
            {
                cout << "* ";
            }
            else
            {
                cout << "  ";
            }
        }
        cout << "\n";
    }

    return 0;
}